InventoryManagement/src/main/java/com/ccs/inventorymanagement/repo/ClothingRepository.java

```diff
@@ -1,10 +1,14 @@
 package com.ccs.inventorymanagement.repo;
 
+import org.springframework.data.r2dbc.repository.Query;
 import org.springframework.data.r2dbc.repository.R2dbcRepository;
 import org.springframework.stereotype.Repository;
+import reactor.core.publisher.Mono;
 
 import java.util.UUID;
 
 @Repository
 public interface ClothingRepository extends R2dbcRepository<ClothingEntity, UUID> {
+    @Query("UPDATE clothing SET status = 'ISSUED' WHERE id = $0")
+    Mono<Void> deleteByID(UUID id);
```

Comment:
You may want to rename this method to let callers know what it really does. Something like "softDelete()" or something like that. 99.9% of the time "delete" in a repo means to remove the row completely. You could even make an update method that takes the enum so you could implement undo operations.